File: 3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.cpp
class Solution {
public:
    bool canAliceWin(vector<int>& nums) {
        int ss=0;
        int ds=0;
        for(int i=0;i<nums.size();i++){
            if (nums[i]<10){
                ss=ss+nums[i];
            }
            else{
                ds=ds+nums[i];
            }
        }
        if (ss!=ds){
            return true;
        }
        else{
            return false;
        }
        
    }
};

File: 3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.py
class Solution:
    def canAliceWin(self, nums: List[int]) -> bool:
        single_digit_sum=0
        double_digit_sum=0
        for i in range(len(nums)):
            if nums[i]<10:
                single_digit_sum+=nums[i]
            else:
                double_digit_sum+=nums[i]
        if single_digit_sum!=double_digit_sum:
            return True
        else:
            return False
        